core/audits/layout-shift-elements.js

```diff
@@ -61,6 +64,8 @@ class LayoutShiftElements extends Audit {
         {nodeCount: clsElementData.length});
     }
 
+    details.metricSavings = {CLS: clsSavings};
```

Comment:
silly question but.... won't this just always equal the total CLS score?

Reply:
This audit is just the top 5 elements

Reply:
_shouldn't_ it equal the total CLS?

Reply:
> don't we want it to equal the total CLS?

IMO yeah

Reply:
Depends how you want to look at it...

On one hand, you can't fix this audit without fixing every LS on the page and therefore fix all CLS.

On the other hand, the report we give you can only help with the elements listed, so the potential savings for that report is limited.

Seems like most of us prefer the former interpretation, it's likely users see it that way as well. Also the "top 5" nature of the audit is just kinda hinted at in the description, not the most obvious. I'm fine changing it.